core/Cargo.toml

```diff
@@ -37,6 +37,14 @@ server = [
 	"tokio",
 ]
 client = ["futures-util"]
+async-client = [
```

Comment:
I had to introduce a feature to avoid bring in `tokio` and some other deps for `http`